<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monitor Configuration</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="main_grid_layout" rowstretch="24,0">
      <item row="0" column="1">
       <widget class="QTabWidget" name="monitor_tab_widget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>430</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="save_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QWidget" name="monitors_area_widget" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
